tasks/main.yml

```diff
@@ -462,14 +462,25 @@
       dest: /usr/local/src/helm
       remote_src: yes
 
-  - name: Copy helm cli to bin directory
+  - name: Copy helm cli to bin directory for amd-64
```

Comment:
a minor nit - usually the arch is referenced as `amd64`

Reply:
ok, will update.